templates/new/config/config.exs

```diff
@@ -12,6 +12,11 @@ config :<%= app_name %>, target: Mix.target()
 
 config :nerves, :firmware, rootfs_overlay: "rootfs_overlay"
 
+# Set the SOURCE_DATE_EPOCH date for reproducible builds.
+# See https://reproducible-builds.org/docs/source-date-epoch/ for more information
+
+config :nerves, source_date_epoch: "<%= source_date_epoch %>"
```

Comment:
No double quotes here since this is an integer?

Reply:
The reason I put double quotes here is because the elixir formatter tries to add the `_` to the number because its so large and it looks weird. for example `"1574186925"` instead of `1_574_186_925`.

Reply:
Good point. I choose string now as well.